lib/validations.js

```diff
@@ -208,6 +220,17 @@ validations.isURL = function isURL(paramName, options, customMessage) {
   );
 }
 
+validations.isPlainObject = function isPlainObjectValidator(paramName, customMessage) {
```

Comment:
another name mismatch here.
`isPlainObjectValidator` should just be `isPlainObject` for consistency.